README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
```

Comment:
Why put this section here? I think the place where it was placed was better than this one.

We want to tell people how to install Spyder first, not how to run it from source.